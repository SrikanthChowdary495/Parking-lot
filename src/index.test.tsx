import React from 'react';
import { render, screen } from '@testing-library/react';
const index = require('./index');

describe('Test index.tsx', () => {-

    it("Should render parking app without crashing", () => {
        expect(
            JSON.stringify(
              Object.assign({}, index, { _reactInternalInstance: 'censored' }),
            ),
          ).toMatchSnapshot();
    });
});