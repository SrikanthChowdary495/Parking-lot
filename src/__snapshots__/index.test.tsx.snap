// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test index.tsx Should render parking app without crashing 1`] = `"{\\"_reactInternalInstance\\":\\"censored\\"}"`;
